### PR TITLE
Increment djsonb version

### PR DESCRIPTION
**Pull request by moradology**

### CODE REVIEW

**Comment by kshepard** [comment]
+1

File: setup.py [hunk 1]
```diff
@@ -13,14 +13,14 @@
     keywords='gis jsonschema',
     packages=find_packages(exclude=['tests']),
     dependency_links=[
-        'https://github.com/azavea/djsonb/tarball/develop#egg=djsonb-0.1.5'
+        'https://github.com/azavea/djsonb/tarball/feature/pattern-search-OR#egg=djsonb-0.1.6'
```
Comment (ddohler) [inline setup.py]: Shouldn't this be `/develop`?

Comment (kshepard) [inline setup.py]: Oh, yes. Good catch @ddohler!

Comment (moradology) [inline setup.py]: My bad on there - had to specify an alternative for provisioning with the in-progress branch to work

**Pull request merged**